apps/glusterfs/app_topology.go

```diff
@@ -0,0 +1,110 @@
+//
+// Copyright (c) 2015 The heketi Authors
```

Comment:
This should be 2018.

Our rule (at least as far as I've been involved in the project) has been that new files should have the current year in the copyright line, but old files do not need to be updated when you touch them.